app/views/taxYearOverview.scala.html

```diff
@@ -22,7 +22,7 @@
 @import views.html.helpers.{breadcrumbHelper, statusHelper}
 @import views.html.templates.main_template
 
-@(taxYear: Int, overview: CalcOverview, transaction: Option[TransactionModel] = None, incomeBreakdown: Boolean = false, deductionBreakdown: Boolean = false)(implicit request: Request[_], messages: Messages, appConfig: config.FrontendAppConfig)
+@(taxYear: Int, overview: CalcOverview, transaction: Option[TransactionModel] = None, incomeBreakdown: Boolean = false, deductionBreakdown: Boolean = false, taxDue: Boolean = false)(implicit request: Request[_], messages: Messages, appConfig: config.FrontendAppConfig)
```

Comment:
taxDue needs adding to the if statement on line 64 as another or case

Reply:
Added